test/validation/api/ipsec/ipsec.c

```diff
@@ -365,9 +365,7 @@ void ipsec_sa_destroy(odp_ipsec_sa_t sa)
 	CU_ASSERT_EQUAL(ODP_IPSEC_OK, odp_ipsec_sa_disable(sa));
 
 	if (ODP_QUEUE_INVALID != suite_context.queue) {
-		do {
-			event = suite_context.dest_queue_deq();
-		} while (event == ODP_EVENT_INVALID);
+		event = suite_context.dest_queue_deq(ODP_TIME_MIN_IN_NS);
```

Comment:
One minute timeout is quite long, especially in the CI which could suffer from it very many times after a faulty commit. I think one second would be plenty, except perhaps in some negative reassembly tests (?).

It would be good to define the timeout as a macro or function so that it could be tweaked later without having to change the code in several places.

Reply:
The commit message still talks about 1 minute timeout.

Reply:
Missed to update the commit message. Will fix this in v9.